src/schemas/components/SchemaFormPropertyUnknown.vue

```diff
@@ -36,6 +36,17 @@
 
   const { property } = useSchemaProperty(() => props.property)
 
+  if (!isDefined(props.property.enum)) {
+    const valueOrDefaultValue = isDefined(props.value) ? props.value : props.property.default
+
+    const json: PrefectKindJson = {
+      __prefect_kind: 'json',
+      value: asJson(valueOrDefaultValue),
+    }
+
+    emit('update:value', json)
```

Comment:
Emitting this basically updates the ui to show a json input rather than this unknown component. This is the same as we were doing before just doing it in setup rather than in the computed.